src/cassy.py

```diff
@@ -214,10 +209,42 @@ def set_user_auth_token(username, password, securitytoken):
     values['username'] = username
     values['password'] = password
     values['securitytoken'] = securitytoken
+    auth_stmt_set = session.prepare(
+        'INSERT INTO '
+        + os.environ.get('DB_USER_TABLE')
+        + ' (username, password, securitytoken)'
+        + ' VALUES(?, ?, ?);'
+    )
     bound = auth_stmt_set.bind(values)
 
     try:
         session.execute(bound)
+        return True
+
+    except Exception as e:
+        raise Exception('Transaction Error Occurred: ' + str(e))
+
+def verify_auth_token(securitytoken):
+    """
+    Verifies session authentication token.
+    """
+    values = {}
+    values['securitytoken'] = securitytoken
+
+    auth_stmt_get = session.prepare(
+        'SELECT username'
+        + ' FROM ' + os.environ.get('DB_USER_TABLE')
+        + ' WHERE securitytoken=?'
+        + ' ALLOW FILTERING;'
+    )
+    bound = auth_stmt_get.bind(values)
+    session.row_factory = named_tuple_factory
+
+    try:
+        rows = session.execute(bound)
```

Comment:
Should probably return True or False based on whether the auth token is verified or not, instead of throwing an Exception. Will need to change this in the calling code as well.